### PR TITLE
Migrate documents to markdown

### PR DESCRIPTION
**Pull request by tewshi**
Part of #54 

All documents have been moved to markdown, so content can be managed from the `/content` folder

Visit the privacy policy, terms of service and impressum pages to test and see changes.

- [x] requires dependency update
- [x] introduces new modules for markdown

### CODE REVIEW

File: components/content/ProseAddress.vue [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+const css = useCssModule();
+</script>
+
+<template>
+  <address :class="css.address">
+    <ContentSlot :use="$slots.default" unwrap="p" />
```
Comment (kelsos) [inline components/content/ProseAddress.vue:7]: nitpick: can we use `useSlots()` instead

File: components/content/ProseH2.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { useRuntimeConfig } from '#imports';
```
Comment (kelsos) [inline components/content/ProseH2.vue:2]: nitpick: this should be auto imported right?

File: components/content/ProseP.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const css = useCssModule();
+</script>
+
+<template>
+  <p
+    :class="[
+      css.text,
+      {
+        [css.small]: small,
+      },
```
Comment (kelsos) [inline components/content/ProseP.vue:20]: I didn't know of this syntax, cool :)

File: package.json [hunk 4]
```diff
@@ -40,6 +40,7 @@
     "zod": "3.20.6"
   },
   "devDependencies": {
+    "@nuxt/content": "^2.4.3",
```
Comment (kelsos) [inline package.json:43]: also small nitpick: can you make sure that we have a strict version?

**Review by kelsos — APPROVED** [review]
@tewshi thanks, looks great. I only have a couple of nitpicks

File: components/content/ProseAddress.vue (hunk 1 above)
Comment (tewshi) [reply components/content/ProseAddress.vue:7]: Okay, I'll check it out.

File: components/content/ProseH2.vue (hunk 2 above)
Comment (tewshi) [reply components/content/ProseH2.vue:2]: yeah, I missed that, will remove.

File: package.json (hunk 4 above)
Comment (tewshi) [reply package.json:43]: sure, will do.
thanks.

File: components/content/ProseH2.vue (hunk 2 above)
Comment (tewshi) [reply components/content/ProseH2.vue]: done

File: components/content/ProseA.vue [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  defineProps<{
+    href: string;
+    target?: string;
+    rel?: string;
+  }>(),
+
+  {
+    href: '#',
+    target: '_self',
+
```
Comment (lukicenturi) [inline components/content/ProseA.vue:12]: remove the blank line

File: components/content/ProseA.vue [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+withDefaults(
+  defineProps<{
+    href: string;
+    target?: string;
+    rel?: string;
+  }>(),
+
```
Comment (lukicenturi) [inline components/content/ProseA.vue:8]: remove the blank line

Comment (tewshi) [reply components/content/ProseA.vue]: done

File: components/content/ProseA.vue (hunk 5 above)
Comment (tewshi) [reply components/content/ProseA.vue]: done

**Pull request merged**